table/run_queries.py

```diff
@@ -27,21 +28,48 @@
 
 client = cloud_bigquery.Client(project=firehook_resources.PROJECT_NAME)
 
+SOURCE_PLACEHOLDER = 'SOURCE'
```

Comment:
Can you make these placeholders more unique (e.g. `$BASE_DATASET$`)? I can see them possibly colliding if we don't pay attention.

Reply:
switched to BASE_DATASET and DERIVED_DATASET. I tried to find a special character like $ that I could use. But everything I tried messed up standard sql syntax highlighting, which would make this file extremely annoying to edit going forward.